modules/RTC/JitsiTrack.js

```diff
@@ -3,9 +3,9 @@ import { getLogger } from '@jitsi/logger';
 import * as JitsiTrackEvents from '../../JitsiTrackEvents';
 import { MediaType } from '../../service/RTC/MediaType';
 import browser from '../browser';
+import Listenable from '../util/Listenable';
```

Comment:
Did you forget to modify the JitsiTrack class definition ? It is supposed to extend Listenable instead of EventEmitter for it to be able to call addCancellableListener.